model-engine/model_engine_server/inference/vllm/Dockerfile.vllm

```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1
-ARG VLLM_VERSION=0.5.3.post1
+ARG VLLM_VERSION=0.6.0
```

Comment:
a `.0` release often has more bugs than a later bugfix release, would it be worth trying to upgrade to [v0.6.1.post2](https://github.com/vllm-project/vllm/releases/tag/v0.6.1.post2) instead?

Reply:
sur